formats/completed_transaction.jsonnet

```diff
@@ -51,6 +52,12 @@
               },
               required: ["url"]
             },
+            {
+              properties: {
+                category: { enum: ["electric_vehicle"] }
+              },
+              required: ["url"]
```

Comment:
Is it required? The [Trello card](https://trello.com/c/FmE5xU1p/1478-3-add-new-bespoke-message-to-specific-completed-transaction-pages#comment-5db70e607675b9862d474390) gave us some text to hardcode. We decided to do it the "proper" way, but do content designers need to set the URL, or can this be entirely hardcoded in https://github.com/alphagov/frontend/pull/2089?

Reply:
The card says we can hard code the text, but I do like doing it the “proper” way.

However, since the text of the link is hard coded in frontend anyway I don’t see any benefit to allowing the URL to be customisable.

Perhaps we should hard code it at the beginning and change it if we need to?

Reply:
Although I see now the way it works in publisher is that the URL input is always visible. Perhaps it’s easiest to be consistent with the others and require the content designer to choose the URL.